Assignment2/DanielMiM-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,49 @@
+{
+    "@context": {
+        "c": "http://example.org",
```

Comment:
Be careful whit prefix definitions, the URI must have a / or # at the end. -> "http://example.org/"